### PR TITLE
#1 review

### PR DESCRIPTION
**Pull request by maflaven**

### CODE REVIEW

File: MyJeopardy.js [hunk 1]
```diff
@@ -74,4 +74,7 @@ async function setupStart() {
  // STEP THREE: What happens when I click the “Restart Game” button?
 
  // Add on click to restart button
-// $("#restart").on("click", setupAndStart);
\ No newline at end of file
+// $("#restart").on("click", setupAndStart);
+
+// start the game on initial page load
+setupStart()
```
Comment (maflaven) [inline MyJeopardy.js:80]: Need to actually start the game.

File: MyJeopardy.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ async function getClues(catId) {
  *   each with a question for each category in a <td>
  *   (initally, just show a "?" where the question/answer would go.)
  */
-async function jeopardyGB () {
+async function populateGameBoard () {
```
Comment (maflaven) [inline MyJeopardy.js:38]: `jeopardyGB` isn't a good function name. Functions should be name according to the actions they perform.

File: MyJeopardy.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ async function getClues(catId) {
         showing: null // displays nothing
     }));
 
-    return {title: cat.title, clues} // returns the title of the category, and category's clues
+    return {title: category.title, clues} // returns the title of the category, and category's clues
```
Comment (maflaven) [inline MyJeopardy.js:28]: You need to run `setupStart` before asking me to review next time. This needed to be changed for it to work.

File: MyJeopardy.js (hunk 1 above)
Comment (mmyatt91) [reply MyJeopardy.js:80]: Okay.

File: MyJeopardy.js (hunk 2 above)
Comment (mmyatt91) [reply MyJeopardy.js:38]: Understood.

File: MyJeopardy.js (hunk 3 above)
Comment (mmyatt91) [reply MyJeopardy.js:28]: Will make the necessary change. And my apologies, I was under the impression that you were checking the accuracy of my code first. But, I understand how difficult that would be, if you don't see that it actually works.

Comment (maflaven) [reply MyJeopardy.js:28]: No worries! And yeah, usually engineers never submit code reviews until their code is working (i.e. all tests pass).

Comment (mmyatt91) [reply MyJeopardy.js:28]: Understood. Makes total sense. So to proceed, should I just do a git pull with my updated/corrected code? Or do I need to follow your pull requests/code review instructions first?

Comment (maflaven) [reply MyJeopardy.js:28]: You should merge this pull request using github. There should be a big green merge button at the bottom. Then pull down the latest `master` on your computer, and then follow what I said about branching as you make more commits.

**Pull request merged**